rsocket/RSocket.cpp

```diff
@@ -18,7 +18,7 @@ folly::Future<std::unique_ptr<RSocketClient>> RSocket::createConnectedClient(
     connectionFactory,
     setupParameters = std::move(setupParameters),
     responder = std::move(responder),
-    keepaliveTimer = std::move(keepaliveTimer),
+    keepaliveInterval = std::move(keepaliveInterval),
```

Comment:
Nit: All these instances of `std::move(keepaliveInternal)` don't need the move.  It's just an integer.